src/file-upload/dropzone.tsx

```diff
@@ -14,7 +14,7 @@ export interface DropzoneElement extends HTMLInputElement {
 }
 
 interface Props extends FileInputProps {
-    children: JSX.Element[];
+    children: React.ReactNode;
```

Comment:
switched because JSX.Element has type error when conditional rendering in this manner:
```
{selected && <div />}
```